frontend/integration-tests/tests/kubevirt/vm.overview.scenario.ts

```diff
@@ -123,4 +122,67 @@ describe('Test vm overview page', () => {
       });
     });
   });
+
+  describe('Test vm overview on project overview page', () => {
+    const newVMDescription = 'edited vm description';
+    beforeEach(async() => {
+      await browser.get(`${appHost}/overview/ns/${testName}`);
+      await isLoaded();
+    });
+
+    it('Check vm details on project overview', async() => {
+      expect(vmView.statusIcon(vmView.statusIcons.off).isPresent()).toBeTruthy();
+      await itemVirtualMachine.click();
+      await isLoaded();
+      await vm.action('Start');
+
+      expect(vmView.vmDetailDesc(testName, vmName).getText()).toEqual(newVMDescription);
+      expect(vmView.vmDetailOS(testName, vmName).getText()).toEqual(basicVmConfig.operatingSystem);
+      expect(vmView.statusIcon(vmView.statusIcons.running).isPresent()).toBeTruthy();
+      expect(vmView.vmDetailTemplate(testName, vmName).getText()).toEqual('openshift/rhel7-generic-small');
+      expect(vmView.vmDetailNamespace(testName, vmName).$('a').getText()).toEqual(testName);
+      expect(vmView.vmDetailBootOrder(testName, vmName).getText()).toEqual(['rootdisk', 'nic0', 'cloudinitdisk']);
+      expect(vmView.vmDetailFlavorDesc(testName, vmName).getText()).toEqual('2 CPU, 4G Memory');
+      expect(vmView.vmDetailIP(testName, vmName).getText()).toContain('10');
+      expect(vmView.vmDetailHostname(testName, vmName).getText()).toEqual(vmName);
+      expect(vmView.vmDetailPod(testName, vmName).$('a').getText()).toContain('virt-launcher');
+      expect(vmView.vmDetailNode(testName, vmName).$('a').getText()).not.toEqual(emptyStr);
+
+      await asyncForEach(exposeServices, async(srv) => {
+        expect(vmView.vmDetailService(testName, srv.exposeName).getText()).toEqual(srv.exposeName);
+      });
+    });
+
+    it('Click vm link', async() => {
+      virtualMachineLink(0).click();
+      expect(browser.getCurrentUrl()).toEqual(`${appHost}/k8s/ns/${testName}/virtualmachines/${vmName}`);
+    });
+
+    it('Click vm status when vm is running', async() => {
+      vmStatusLink(0).click();
+      expect(browser.getCurrentUrl()).toContain('pods');
```

Comment:
It would be better to extend DetailView with a method that returns name of active tab. It has a class `co-m-horizontal-nav-item--active`.

Reply:
The intention here is clicking the running link and it jumps to vm pod page.

Reply:
I see, so I would still do it for checking that we landed on `Events` tab, instead of checking the url.

Reply:
Add a check for this now.